easybuild/easyconfigs/g/GULP/gulp-6.0-mkgulp.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/gulp-6.0.org/Src/mkgulp b/gulp-6.0/Src/mkgulp
+index 28afb80..84549b0 100755
+--- a/gulp-6.0.org/Src/mkgulp
++++ b/gulp-6.0/Src/mkgulp
+@@ -170,7 +170,7 @@ fi
+ #  Include file directories
+ #
+ #--USER--Start
+-echo 'INC=-I.. -I/usr/local/include/' >> makefile
++echo 'INC=-I.. -I${EBROOTFFTW}/include -I/usr/local/include/' >> makefile
+ #--USER--End
+ #  If using Macports then use the following instead
+ #--USER--Start
+@@ -202,19 +202,19 @@ case $compiler in
+                echo 'SLIBS= ' >> makefile
+ #  If there are optimised maths libraries available replace the routines below
+ #--USER--Start
+-               echo 'MLIBS=lapack.o blas.o' >> makefile
++               echo 'MLIBS=-L${LD_LIBRARY_APTH} -lscalapack -lopenblas ' >> makefile
+ #--USER--End
+            else
+ # NB: For gfortran v10 then uncomment these lines and comment the ones immediately below!
+-#               echo 'RUNF90=mpif90 -fallow-argument-mismatch ' >> makefile
+-#               echo 'RUNCC=mpif90 -fallow-argument-mismatch ' >> makefile
+-               echo 'RUNF90=mpif90 ' >> makefile
+-               echo 'RUNCC=mpif90 ' >> makefile
++               echo 'RUNF90=mpif90 -fallow-argument-mismatch ' >> makefile
++               echo 'RUNCC=mpif90 -fallow-argument-mismatch ' >> makefile
++#               echo 'RUNF90=mpif90 ' >> makefile
++#               echo 'RUNCC=mpif90 ' >> makefile
+ #  Provide the location and name of Scalapack, Blacs, Pblas, Lapack and Blas libraries.
+ #  On the Mac the following is sufficient when using Macports:
+ #--USER--Start
+-               echo 'SLIBS=-L/opt/local/lib -lscalapack ' >> makefile
+-               echo 'MLIBS= ' >> makefile
++               echo 'SLIBS=-L${LD_LIBRARY_PATH} -lscalapack ' >> makefile
++               echo 'MLIBS=-L${LD_LIBRARY_APTH} -lscalapack -lopenblas ' >> makefile
+ #--USER--End
+            fi
+            ;;
+@@ -321,12 +321,13 @@ then
+ else
+   echo 'FDEFS=-DFFTW3 ' >> makefile
+   if [ "$parallel" = "" ]
++  then
+ #--USER--Start
+-    echo 'FLIBS=-L/usr/local/lib/ -lfftw3.a' >> makefile
++    echo 'FLIBS=-L${LD_LIBRARY_PATH} -lfftw3' >> makefile
+ #--USER--End
+-  then
++  else
+ #--USER--Start
+-    echo 'FLIBS=-L/usr/local/lib/ -lfftw3 -lfftw3_mpi' >> makefile
++    echo 'FLIBS=-L${LD_LIBRARY_PATH} -lfftw3 -lfftw3_mpi' >> makefile
```

Comment:
and here as well

Reply:
I tried to follow your suggestions. I removed the `LD_LIBRARY_PATH` etc but I was unable to follow what you suggested regarding using `LIBFFT` etc. 
I have added the author and description in the patch file and noticed the `foss-2020a` should not have worked as the patch file was wrong for `gcc-9.3.0`. So I added a 2nd patch file for that version. I also noticed a missing flag for `foss` so I added that one to have a parallel build. 
Let me know how to deal with the patch files if you are not happy with them.  :-)